content/ch-applications/vqe-molecules.ipynb

```diff
@@ -158,7 +158,7 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 1,
+   "execution_count": 9,
```

Comment:
Can you change these execution counts by rerunning all the cells from starting?
Also, you haven't executed some of the last cells. Please rerun them too.